src/features/classification/security-controls/utils.js

```diff
@@ -10,58 +10,63 @@ import downloadRestrictionsMessageMap from './downloadRestrictionsMessageMap';
 import messages from './messages';
 import { ACCESS_POLICY_RESTRICTION, DOWNLOAD_CONTROL, LIST_ACCESS_LEVEL, SHARED_LINK_ACCESS_LEVEL } from '../constants';
 
+type MessageItem = {
+    message: ?MessageDescriptor,
```

Comment:
I think this should be in `flowTypes` and `message` should be required as mentioned above